### ISSUE TITLE
Posting comments on github sometimes doesn't show the comment without reload

### ISSUE THREAD

**Issue by bbondy** [brave/browser-laptop#402]
1. Add a comment on a commit.

Actual results: Sometimes doesn't load your comment inline.  You need to reload the page to see it.

Expected results: Loads your comment wihtout reloading the page.

**Comment by bbondy** [comment]
Also happens for issues

**Comment by diracdeltas** [comment]
maybe related, but when i click "add a line note" on github, nothing happens

**Comment by bbondy** [comment]
Are you using a version below  Electron: 0.36.5, from debug maybe?  
In 0.7.11 we show the info in the about dialog by the way.
Reason I ask is because this was a known issue in 0.36.3 and it went away for me in 0.36.5.

**Comment by diracdeltas** [comment]
nope, using 0.7.11

**Comment by bbondy** [comment]
ok just noting that it's possible to use app version 0.7.11 with electron 0.36.3 if node_modules is old and node_modules didn't get the new electron-prebuilt.

**Comment by bbondy** [comment]
(Only from debug env though)

**Comment by diracdeltas** [comment]
```
~/r/browser-electron (master) [1]> npm list | grep electron
brave@0.7.11 /Users/yan/repos/browser-electron
├─┬ electron-builder@2.6.0
├─┬ electron-installer-squirrel-windows@1.2.2
├── electron-localshortcut@0.6.0
├─┬ electron-packager@5.2.0 (git://github.com/brave/electron-packager.git#544fa851b3cc7430f7570e1fe7c949bbcb9c7283)
│ ├─┬ electron-download@1.4.1
├─┬ electron-prebuilt@0.36.5 (git://github.com/brave/electron-prebuilt.git#80392709b5a3221b22d63006041245602e1545be)
│ └─┬ electron-download@1.4.1 (git://github.com/brave/electron-download.git#fdbd5be0614c84456b45ec59e79da58e69d9d2c2)
├─┬ electron-rebuild@1.1.2
├── electron-squirrel-startup@0.1.4
│ ├─┬ electron-chromedriver@0.36.0
```

should be 0.36.5?

**Comment by diracdeltas** [comment]
restarted brave and now adding line notes seems to work

**Comment by bbondy** [comment]
seems like we can get in some broken site which breaks content sometimes.  Check console next time. Thanks.

**Comment by bbondy** [comment]
test

**Comment by bbondy** [comment]
This is fixed in 0.36.7 which will be released with 0.7.12

**Comment by diracdeltas** [comment]
can confirm that all of the above are fixed now

**Comment by diracdeltas** [comment]
actually it seems to be fixed in this tab but not in a second tab i just opened. also in the second tab, tagging users (via @) is broken. using Electron: 0.36.7

**Comment by diracdeltas** [comment]
The error that happens in tabs where github AJAX-y actions are broken is
```
Uncaught TypeError: Cannot read property 'isMinimized' of null
```
in Brave.app/Contents/Resources/atom.asar/renderer/lib/override.js:218

**Comment by bbondy** [comment]
Looks like remote.getCurrentWindow() returns null sometimes.  We could do a null check and return true or false? Seems like something else is going on though.

```
// Make document.hidden and document.visibilityState return the correct value.
Object.defineProperty(document, 'hidden', {
  get: function() {
    var currentWindow;
    currentWindow = remote.getCurrentWindow();
    return currentWindow.isMinimized() || !currentWindow.isVisible();
  }
});
```

**Comment by bbondy** [comment]
Opening until this is fixed, we should try to fix this for the next release.

**Comment by bridiver** [comment]
where did you test that code snippet @bbondy ?

**Comment by bridiver** [comment]
if that's the issue this max fix itself when we remove node because that code won't run anymore

**Comment by bbondy** [comment]
Fixed in 0.7.13 which is coming out soon.
